modules/components/src/Aggs/RangeAgg.js

```diff
@@ -114,42 +114,40 @@ class RangeAgg extends Component {
                 <span className={`arrow ${isCollapsed && 'collapsed'}`} />
               )}
             </div>
-            {!isCollapsed &&
-              min !== null &&
-              max !== null && (
-                <div className="range-wrapper">
-                  <div className="unit-wrapper">
-                    {unit &&
-                      SUPPORTED_CONVERSIONS[convert().describe(unit).measure]
-                        .map(x => convert().describe(x))
-                        .map(x => ({ ...x, active: x.abbr === displayUnit }))
-                        .map(({ abbr, plural, active }) => (
-                          <span key={abbr}>
-                            <input
-                              type="radio"
-                              id={abbr}
-                              value={abbr}
-                              checked={active}
-                              onChange={e =>
-                                this.setState({ displayUnit: e.target.value })
-                              }
-                            />
-                            <label htmlFor={abbr}>{plural}</label>
-                          </span>
-                        ))}
-                  </div>
-                  <div className="input-range-wrapper">
-                    <InputRange
-                      minValue={min}
-                      maxValue={max}
-                      value={value}
-                      formatLabel={this.formatRangeLabel}
-                      onChange={x => this.setValue(x)}
-                      onChangeComplete={this.onChangeComplete}
-                    />
-                  </div>
+            {[!isCollapsed, min != null, max != null].every(Boolean) && (
```

Comment:
I'm going to be annoying and as for no `!=`. I know it does what you need, but it's hard to know in the future if it's on purpose or not :D 
maybe use `isNil` from lodash instead?

Reply:
ja thought this might come up, agree it seems like something that will be "corrected" in the future, will do

Reply:
thanks! :D